webpack.config.js

```diff
@@ -0,0 +1,46 @@
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const buildName = 'keen-explorer';
+
+const devModule = {
+  entry: ['./client/js/app/app.js', './client/styles/base.less'],
+
+  output: {
+    filename: `${buildName}.js`,
```

Comment:
When we do a webpack production build we'll want to add a `.min` to this filename. The easiest way I've found to do this is to just check the environment and manually add a `.min` when building for production. Example from statics: https://github.com/keenlabs/keen-statics/blob/master/webpack.config.js#L9